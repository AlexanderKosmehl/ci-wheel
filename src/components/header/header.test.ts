import { describe, expect, it } from 'vitest';
import generateHeader from './header';

describe('generateHeader', () => {
  const header = generateHeader();

  it('generates component correctly', () => {
    expect(header).toMatchSnapshot();
  });
});
