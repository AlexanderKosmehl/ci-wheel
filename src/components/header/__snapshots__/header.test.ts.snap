// Vitest Snapshot v1

exports[`generateHeader > generates component correctly 1`] = `
<header
  class="_header_503ab9"
>
  <div
    class="_logoTitleWrapper_503ab9"
  >
    <img
      class="_headerLogo_503ab9"
      src="/src/assets/ci-logo.png"
    />
    <h1
      class="_headerTitle_503ab9"
    >
      Wheel
    </h1>
  </div>
  <div
    class="_menuButtonWrapper_503ab9"
  >
    <button
      class="_button_7c1f96 _sidebarToggle_503ab9"
    >
      <img
        class="_icon_7c1f96"
        src="/src/assets/menu-icon.svg"
      />
    </button>
    <button
      class="_button_7c1f96 _sidebarToggle_503ab9 _hidden_503ab9"
    >
      <img
        class="_icon_7c1f96"
        src="/src/assets/arrow-back-icon.svg"
      />
    </button>
  </div>
</header>
`;
